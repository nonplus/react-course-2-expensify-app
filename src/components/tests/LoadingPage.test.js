import { shallow } from "enzyme";
import React from "react";
import { LoadingPage } from "../LoadingPage";

describe("LoadingPage", () => {
  it("should render page correctly", () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
