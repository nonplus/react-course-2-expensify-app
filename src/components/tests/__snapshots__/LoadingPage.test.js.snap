// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingPage /> should render page correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
